snippets/saved_replies.md

```diff
@@ -0,0 +1,103 @@
+2. Acknowledge their need for help.
+Unless it's a PR for a typo, most likely they actually feel a need for whatever they are asking about.
+2. Give context.
+Considerations, better places to go with this, etc. If it's anything we have docs on, enable them to come up to speed with the factual basis we have.
+3. Provide a course of action.
+At the least, if we close anything quickly, we should tell them they can ask to re-open, or where else to take this. Again, take into account that there's a need they feel they need to address, somehow.
+
+It's not always feasible, or even possible, to fill out the XXX. Still, we should try to.
+
+
+
+# Issues
+
+
+## User Support
+
+Thanks for your report. It looks as if this is actually a question about usage and not development.
+
+To make your question, and all replies, easier to find, we suggest you move this over to our [user mailing list](https://groups.google.com/forum/#!forum/prometheus-users); for more interactive help, but no long-term archive, or search function, join or our [IRC channel](https://webchat.freenode.org/?channels=prometheus), #prometheus on irc.freenode.net.
+
+If you think this is not purely a support question, feel free to comment in here or take the underlying issues to our [developer mailing list](https://groups.google.com/forum/#!forum/prometheus-developers).
+
+Once your questions have been answered, please add a short line pointing to relevant replies in case anyone stumbles here via a search engine in the future.
+
+All that being said, in your specific case, it might make sense to / please look at / our reasoning can be found at XXX
+
+
+# PRs
+
+
+## DCO
+
+Thanks for your PR. For legal reasons, we require that all commits are signed with a [DCO](https://developercertificate.org/) before we can merge them.
+
+This means that the last line of your commit message should read like:
+
+`Signed-Off-By: Your Name <your@email.address>`
+
+If you are using GitHub through the web interface, it's quickest to close this PR and open a new one with the appropriate line.
+
+If you are using Git on the command line, it is probably quickest to amend and force push. You can do that with
+````
+git commit --amend --signoff
+git push -f $remote $remote_branch_for_pr
+````
+
+As always, be careful when force-pushing.
+
+
+## Moratorium on integrations (SD & Alertmanager)
+
+Thanks for suggesting this change. Unfortunately, we are currently not accepting new integrations.
+
+We can only provide the stability and performance we want to provide if we can test integrations in an automated and scalable fashion. For this reason, we are suggesting people integrate with the help of our generic interfaces by either integrating natively or putting a rewriting proxy in the middle. We would be more than happy to list this on our integrations page.
```

Comment:
The whole point of the moratorium is that we're not accepting native integrations

Reply:
I meant to say that we could list "Foobar is [natively] accepting Alertmanager alerts". I can see how this can be confusing, though.

Reply:
And SD doesn't involve a proxy, it's dropping files in. I'd end the sentence at "interfaces".

Reply:
This is still incorrect

Reply:
Done.